internal/env/parser_test.go

```diff
@@ -0,0 +1,129 @@
+package env
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/mainflux/mainflux/pkg/errors"
+
+	"github.com/mainflux/mainflux/internal/clients/grpc"
+	"github.com/mainflux/mainflux/internal/server"
+	"github.com/stretchr/testify/assert"
+)
+
+var errNotDuration error = errors.New("unable to parse duration")
+
+func TestParse(t *testing.T) {
```

Comment:
Split gRPC, HTTP, and other configs to separate cases lists, or even separate tests.